src/test/java/com/thirdcc/webapp/web/rest/EventResourceIT.java

```diff
@@ -189,6 +210,7 @@ public void createEventWithExistingId() throws Exception {
 
         // An entity with an existing ID cannot be created, so this API call must fail
         restEventMockMvc.perform(post("/api/events")
+            .with(user("user").password("user").roles("ADMIN"))
```

Comment:
Will it be better if this line same as line 167 where all params are admin
.with(user("admin").password("admin").roles("ADMIN"))